daq_macros.py

```diff
@@ -1655,12 +1715,13 @@ def reprocessRaster(rasterReqID):
   procFlag = 1
   
   for i in range(len(rasterDef["rowDefs"])):
+    currentRow = rasterDef["rowDefs"][i]
     time.sleep(0.5) 
```

Comment:
Why is this sleep in the middle of a processing loop?

Reply:
I am guessing this is to prevent all of the row reprocessing from being dumped onto processing nodes at once, which would overwhelm them. fortunately or unfortunately, this is not one of the parts that I changed...